test/integration/features/cmd-addons.feature

```diff
@@ -40,25 +41,14 @@ Feature: Addons command and its subcommands
       Unable to remove addon 'anyuid'. No anyuid.addon.remove file is found.
       """
 
-  Scenario: Installing default add-ons manually
-     When executing "minishift addons install --defaults" succeeds
-     Then stdout should contain
-      """
-      Default add-ons 'anyuid, admin-user, xpaas, registry-route' installed
-      """
-     When executing "minishift addons list" succeeds
-     Then stdout should contain "admin-user"
-     Then stdout should contain "anyuid"
-     Then stdout should contain "registry-route"
-     Then stdout should contain "xpaas"
-
```

Comment:
@LalatenduMohanty Why did we remove this scenario instead of update, does it already part of somewhere else?